tests/infer/test_inference.py

```diff
@@ -371,8 +374,8 @@ def model():
             mu_latent = pyro.sample("mu_latent",
                                     dist.Normal(self.mu0, torch.pow(self.tau0, -0.5)))
             sigma = torch.pow(self.tau, -0.5)
-            pyro.observe("obs0", dist.LogNormal(mu_latent, sigma), obs=self.data[0])
-            pyro.observe("obs1", dist.LogNormal(mu_latent, sigma), obs=self.data[1])
+            pyro.observe("obs0", dist.LogNormal(mu_latent, sigma), obs=self.data[0].squeeze())
```

Comment:
This test fails to meet the threshold if we use a vectorized "obs" inside an `iarange`. 

@martinjankowiak, @frtizo - Is that expected? Is there a reason why the two `obs` are observed in separate statements?

Reply:
Just checked with @martinjankowiak that this works as expected inside `iarange`, so I am not sure why I was seeing a difference. Will make the change and update.

Reply:
@martinjankowiak can we just delete this flaky expensive test?

Reply:
yes we can. although maybe keep the transformed_distribution bit?